module-1/m14-wev-dev-environment/muder-mystery.md

```diff
@@ -4,4 +4,153 @@ author: Tuan Hoang
 description: solutions for full stack web assignments
 ---
 
-[Back to README](../README.md)
+
+# Murder Mystery
+Init: Open file README.md
+Open your terminal
+
+ 
+#### 00 
```

Comment:
Please use markdown linter. 
number of `#` must follow order #, ##, ###, #### .
Step should have work step then number